app/controllers/accountants_controller.rb

```diff
@@ -5,26 +5,33 @@ class AccountantsController < ApplicationController
   before_action :find_patient, only: [:edit]
 
   def index
-    patients = pledged_patients.includes(:clinic)
-    @patients = paginate_results(patients)
+    generate_base_patients
   end
 
   def search
+    generate_base_patients
+      
     have_search = params[:search].present?
     have_clinic = params[:clinic_id].present?
 
     if have_search || have_clinic
-      partial = Patient.where(pledge_sent: true)
-                       .includes(:clinic)
-                       .includes(:fulfillment)
-                       .order(pledge_sent_at: :desc)
+      partial = pledged_patients
 
       partial = partial.where(clinic_id: params[:clinic_id]) if have_clinic
-      partial = partial.search(params[:search]) if have_search
+      partial = partial.search(params[:search], search_limit: nil) if have_search
 
       @results = partial
     else
-      @results = pledged_patients
+      @results = pledged_patients.where(initial_call_date: 6.months.ago..)
+    end
+  
+    @results = paginate_results @results
+
+    if @results.length != @patients.length
+      extra_display = t('accountants.extra_count', count: @patients.length,
+                                                   entry: t('common.patient').pluralize(@patients.length))
+      @result_count_extra = " (#{extra_display})"
+      @entry_name = t('accountants.results')
     end
```

Comment:
I think I'm coming to the conclusion that while I like the (out of X patients) theoretically, it just isn't a reliable number we can pull as we adjust the results through searching At this point I'm thinking we just nix the Extra Display rather than keep spinning on this otherwise awesome PR. Thoughts? @elimbaum @colinxfleming?

Reply:
this is so weird! let me take another look at this - i can have an answer to you this weekend. what we're searching for shouldn't have a bearing on the total number of patients...!

also I think I should change the language. My `(out of X patients)` is dangerously similar to kaminari's `of X in total`. I'm realizing now you could end up in a situation that read something like `Displaying patients 1 - 25 of 37 in total (out of 47 patients total)` which is rather confusing!

also thank you for your diligence on this! i definitely need to do a leeeeeetle bit more manual testing and not rely entirely on unit tests! or, i need better unit tests ;)

Reply:
Enormously in favor of nuking the Extra display - low-to-nil value add to an accountant + high code complexity = great target for removal

Reply:
Totally fair. do not like the below... in the future if we needed we could possibly do language like (`...out of X patients in database`) 

<img width="538" alt="weird display" src="https://user-images.githubusercontent.com/7085805/123518593-96d4a200-d674-11eb-8924-2b14b8d12b8a.png">

Reply:
@lomky, did some digging, and the bug you observed is caused by Kaminari objects' `.count` method returning the number of elements on the current page, rather than the number of elements in the entire set. For that, you have to use `.total_count`; with that change I was able to replicate and then fix the issue you observed. However, agree with your instinct; I'll remove it.